Add mirrord vs TP blog post (#56)
metalbear-co/metalbear.co: content/en/blog/mirrord-vs-telepresence/index.md

4. As you might have guessed, mirrord **supports traffic mirroring**. It can still intercept traffic like Telepresence does, but if you want to leave the remote process completely untouched, mirrord can also just duplicate its traffic and send a copy to you. The original requests are handled by the remote service, so people accessing it are completely unaware that you’re using mirrord to debug it.
5. As opposed to Telepresence which works at the network level, mirrord works at the pod level. This means it also **supports pods that aren’t exposed through a service**, and gives your local code access to anything the pod can access, including **components outside the cluster**. It also natively supports environments with service mesh like Linkerd or Istio.
6. Because the mirrord client works at the process level, you can easily **configure exactly what’s executed remotely and what stays local**. For example, you could make all the components of the remote cluster accessible to your local process, but have it write to the database locally; or have it read files from the local file system, except a few specific files that you want to get from the remote pod. 

> 7. mirrord is a tool for developers, and as such, in addition to a CLI tool, it also comes as an **IDE plugin for either VS Code or the IntelliJ IDEs**. This is the most popular way to use mirrord, where it naturally fits into the existing development workflow.

Reviewer: ```suggestion
7. mirrord is a tool for developers, and as such, in addition to a CLI tool, it also comes as an **IDE plugin for both VS Code and the IntelliJ IDEs**. This is the most popular way to use mirrord, where it naturally fits into the existing development workflow.
```
